src/steps/memberSteps.ts

```diff
@@ -30,9 +31,9 @@ Given("user insert personal data", async function () {
   );
 });
 
-When("the user clicks on {string} button", async function (string) {
-  await memberPage.clickSaveButton();
-});
+// When("the user clicks on {string} button", async function (string) {
```

Comment:
please check if this step is used anywhere in tests. if not, please remove it

Reply:
Removed.